src/commands/app/add/event.js

```diff
@@ -29,27 +31,63 @@ class AddEventCommand extends AddCommand {
     if (entries.length > 1) {
       this.error('Please use the \'-e\' flag to specify to which implementation you want to add events to.')
     }
+
     const configName = entries[0][0]
     const config = entries[0][1]
     const actionFolder = path.relative(config.root, config.actions.src)
     const configData = this.getRuntimeManifestConfigFile(configName)
 
-    const env = yeoman.createEnv()
-    // by default yeoman runs the install, we control installation from the app plugin
-    env.options = { skipInstall: true }
-    const eventsGen = env.instantiate(generators['add-events'], {
-      options: {
-        'skip-prompt': flags.yes,
-        'action-folder': actionFolder,
-        'config-path': configData.file,
-        'full-key-to-manifest': configData.key,
-        // force overwrites, no useless prompts, this is a feature exposed by yeoman itself
-        force: true
+    const templateOptions = {
+      'skip-prompt': false,
+      'action-folder': actionFolder,
+      'config-path': configData.file,
+      'full-key-to-manifest': configData.key
```

Comment:
if the events template API is still open for changes I would use `full-key-to-runtime-manifest` in the new template. Otherwise nvm

Reply:
I let it be consistent with other templates where full-key-to-manifest meant runtimeManifest.

Reply:
also, the ActionGenerator which is extended by the EventsGenerator expects the full-key-to-manifest

Reply:
ok then nvm 👍